ledger/src/blockstore_processor.rs

```diff
@@ -1588,19 +1587,23 @@ fn load_frozen_forks(
                 slot,
             );
 
-            process_next_slots(
-                &bank,
-                &meta,
-                blockstore,
-                leader_schedule_cache,
-                &mut pending_slots,
-            )?;
-
-            if slot >= halt_at_slot {
```

Comment:
Important note here; we were creating the new Bank(s) in `process_next_slots()`, but not adding them `BankForks`. So, these `Bank`s were essentially orphaned, and explains why we'd try to create a Bank for the slot a second time when `ReplayStage` fired up

Reply:
Using `halt_at_slot` is a change in behavior in v1.13 since we started grabbing new shreds while we're processing the local ledger (ie prevents load_frozen_forks() from running indefinitely). So, this is not an issue in v1.13 and I'll be removing the backport label